.editorconfig

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
nice addition ig
is that empty line some sort of convention?
if not please remove